### DISCUSSION TITLE
How to follow on mobile in more seamless way?

### DISCUSSION THREAD

**Discussion by MiKom** [mastodon/mastodon#18175] [Q&A] [answered]
Hello,

on my iPhone (that has the mastodon app installed) I recently tried to follow an account that advertised its mastodon link in a tweet on twitter (https://twitter.com/mozthunderbird/status/1519535865365553152)

When I clicked the link, the profile opened. The follow button on the profile opend a web-based workflow for following that would require me to log in to my mastodon account in this web session.

Is there some way where the mobile client could intercept the follow link and let it follow in the app?

If not, maybe it would make sense to modify the profile page so that if it recognizes we're on mobile, it makes the follow button lead to an own url scheme that could be intercepted by the native app?

**Comment by Gargron** [comment, marked as answer]
I think at least on iOS we could do with it with Smart App Banners...

**Reply by MiKom** [reply]
Hmm. Having now read about it seems that it could work. I'll file an issue for it. Thanks for guidance, and thanks for Mastodon!

**Reply by MiKom** [reply]
I filed the issue: #18183
